### PR TITLE
IvyMooMoo - HoloID

### PR DESCRIPTION
**Pull request by DarkWolfInsanity**
Custom Item for IvyMooMoo and kit for spawning

Ivy's HoloID
id

Kit includes:
- Ivy Baladeva's HoloID
- A Vanilla Cake
- A Box of Cigarettes
- A Box of Matches
- A White Beret

// The ID alone was unable to spawn being put in the custom_items.txt file. It required being put in a kit. //

### CODE REVIEW

File: code/modules/vore/fluffstuff/custom_boxes_vr.dm [hunk 1]
```diff
@@ -140,3 +140,16 @@
...
+
+//ivymoomoo:Ivy Baladeva
+/obj/item/weapon/storage/backpack/messenger/sec/fluff/ivymoomoo
+	name = "Ivy's Courier"
+	desc = "A bag resembling something used by college students. Contains items for ''MooMoo''."
+
+	New()
+		..()
+		new /obj/item/clothing/head/beretg(src)
+		new /obj/item/weapon/card/id/centcom/fluff/ivyholoid(src)
```
Comment (Cameron653) [inline code/modules/vore/fluffstuff/custom_boxes_vr.dm]: This line needs to be  changed to
new /obj/item/weapon/card/id/fluff/ivyholoid(src)
so it'll compile correctly & work correctly.

**Comment by DarkWolfInsanity** [comment]
I'll have to recheck it all since I believe it does conflict with the master branch now, but otherwise it got accepted on the forums.

**Comment by SpadesNeil** [comment]
@Arokha this is probably fine now, right?

**Comment by SpadesNeil** [comment]
Meh.

**Pull request merged**